languages.toml

```diff
@@ -335,18 +335,18 @@ name = "awk"
 source = { git = "https://github.com/Beaglefoot/tree-sitter-awk", rev = "a799bc5da7c2a84bc9a06ba5f3540cf1191e4ee3" }
 
 [[language]]
-name = "protobuf"
+name = "proto"
```

Comment:
the name of the language can remain unchanged if you set a key `grammar = "proto"`. The `grammar` name needs to match the `name` key in `grammar.js` (https://github.com/sdoerner/tree-sitter-proto/blob/778ab6ed18a7fcf82c83805a87d63376c51e80bc/grammar.js#L16) but the language name can be different